scripts/postinstall.sh

```diff
@@ -4,7 +4,8 @@ set -e
 
 if [ -n "$SKIP_POSTINSTALL_DEV_SETUP" ]; then
   echo "Skipping development setup."
-
+elif [[ $NPM_CONFIG_CACHE =~ "renovate" ]]; then
```

Comment:
The environment variable is set by Renovate and is "NPM_CONFIG_CACHE": "/tmp/renovate-cache/others/npm" anywhere I looked.

Reply:
Do we need to skip the entire `postinstall.sh` script to be executed or only certain parts?

For instance, the `yarn prettier` command is still executed here.

Reply:
If you know and are willing to debug which parts can be included and excluded we can skip those. Still I think it's besides the point.

I didn't intend this to be a permanent solution. As noted in the description. Ideally Renovate should be able to skip the entire post install as it does with older yarn versions or NPM. It's not relevant for Renovate to run it.